processor/tailsamplingprocessor/README.md

```diff
@@ -1,6 +1,8 @@
-# Tail Sampling Processor
-
-Supported pipeline types: traces
+| Tail Sampling Processor  |               |
```

Comment:
I like keeping the processor title as a top-level header.  Could we get away without putting it in the table?

Reply:
Done